rct-graphics-helper/properties/general_properties.py

```diff
@@ -94,6 +94,24 @@ class GeneralProperties(bpy.types.PropertyGroup):
         description="Which color groups to dither to. Recolorables will be excluded from this palette when used to avoid conflicts.",
         size=len(defaults))
 
+    dithering_mode = bpy.props.EnumProperty(
+        name = "Dithering Mode",
```

Comment:
If we're only having FloydSteinberg then there is no need for a dropdown anymore